### PR TITLE
EICNET-1161: Implement the trigger for all activity items

### PR DESCRIPTION
**Pull request by 4hardfire**
### Implementations

- Created missing fields in message type "STREAM :: MT36 :: Photo album insert/update";
- Trigger message creation "STREAM :: MT36 :: Photo album insert/update" when create/update a gallery;

### Tests

- [x] Create a new group and publish it
- [x] Create a new group content "Gallery" and check the "Post activity" checkbox
- [x] Go to `/admin/content/message` and check if the message has been created
- [ ] Create a new group content "Video" and check the "Post activity" checkbox
- [ ] Go to `/admin/content/message` and check if the message has been created

### Questions

- I suppose the message type "STREAM :: MT36 :: Photo album insert/update" comes from the old D7 version. Should we replace the machine name and labels with new ones according to the Content Type name (Gallery)?

**Solved**: created new message type.

- Currently the only group content that is missing the Post activity feature is the Video. There is already a message type "STREAM :: MT33 :: Photo insert/update" that is not addressed to any content type. Should we take this one or create a new one with a different name?

**Solved**: created new message type.

### CODE REVIEW

File: lib/modules/eic_messages/src/Service/GroupContentMessageCreator.php [hunk 1]
```diff
@@ -85,28 +84,54 @@ public function createGroupContentActivity(
...
+  }
+
+  /**
+   * Gets activity message template for a given entity.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity object.
    *
    * @return string
+   *   The message template name.
```
Comment (korhankale) [inline lib/modules/eic_messages/src/Service/GroupContentMessageCreator.php:121]: Just a quick note with #498 this method has been moved into another object, before merging this we'll have to add the gallery template there (cfr: https://github.com/EIC-EA/EIC-community-D8/blob/0184a0b0f4294903b9f19295af105b25dcf93cf5/lib/modules/eic_messages/src/Util/ActivityStreamMessageTemplates.php)

Comment (korhankale) [reply lib/modules/eic_messages/src/Service/GroupContentMessageCreator.php:121]: #489 has been merged

Comment (4hardfire) [reply lib/modules/eic_messages/src/Service/GroupContentMessageCreator.php]: Done!

**Review by lramarojaona — APPROVED** [review]
Looks good!

**Comment by lramarojaona** [comment]
@4hardfire , works as expected!

**Review by lramarojaona — APPROVED** [review]

**Pull request merged**